HW04/tim_bulat/lib/student.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative 'person'
+
+class Student < Person
+  def initialize(name:, surname:)
+    super(name: name, surname: surname)
+    @homeworks = []
+  end
+
+  def add_answer!(homework, answer)
+    homework.answer = answer
+  end
+
+  def to_check!(homework)
+    homework.is_check = true
+    homework.mentor.notes << "Student #{name} send homework #{homework.title} to check\n" \
+                              "Answer:#{homework.answer}\n"
+  end
+
+  def homeworks
+    @homeworks.each do |homework|
+      print "Homeworks:\n
+            Title: #{homework.title}\n
+            Description:#{homework.description}\n"
+    end
+  end
```

Comment:
can we use another name to split responsibility? for example
```suggestion
  def show_homeworks
    @homeworks.each do |homework|
      print "Homeworks:\n
            Title: #{homework.title}\n
            Description:#{homework.description}\n"
    end
  end
```